packages/react-dom/src/client/ReactDOM.js

```diff
@@ -108,10 +107,10 @@ function createPortal(
   container: Container,
   key: ?string = null,
 ): React$Portal {
-  invariant(
-    isValidContainer(container),
-    'Target container is not a DOM element.',
-  );
+  if (!isValidContainer(container)) {
```

Comment:
I think this is showing up because Andrew recently changed this and something about the base revision I'm on. Let me rebase and see if it goes away. I didn't do this though